<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QLabel" name="plan_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>План</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="product_name_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Наименование продукции</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="plan_1_lineedit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="scaner_status_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Связь</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="4">
     <widget class="QPushButton" name="make_template_pushbutton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Загрузить шаблон</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="2">
     <widget class="QLineEdit" name="product_1_name_lineedit">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>sir Kavkazskiu</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="current_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Факт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="line_1_status_checkbox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QPushButton" name="line_1_start_pushbutton">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="line_number_1_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Линия №1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QLabel" name="current_1_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="status_finish_label">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(141, 255, 255);</string>
      </property>
      <property name="text">
       <string>Завершён</string>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <widget class="QCheckBox" name="line_1_status_finish_checkbox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
